test/lolex-test.js

```diff
@@ -2053,6 +2053,17 @@ describe("lolex", function () {
                 clock.runAll();
                 assert(!called);
             });
+
+            it("passes arguments when installed - GitHub#122", function () {
+                var clock = this.clock = lolex.install();
```

Comment:
i think he meant on line 2058
`var clock = this.clock = lolex.install();`
could be changed to:
`var clock = lolex.install();`